test/extended/operators/olm.go

```diff
@@ -151,6 +151,41 @@ var _ = g.Describe("[sig-operator] OLM should", func() {
 	})
 })
 
+func hasRedHatOperatorsSource(oc *exutil.CLI) (bool, error) {
+	spec, err := oc.AsAdmin().Run("get").Args("operatorhub/cluster", "-o=jsonpath={.spec}").Output()
+	if err != nil {
+		e2e.Logf("Error reading operatorhub spec: {}", spec)
```

Comment:
wrap error and remove log line

Reply:
Reworked this to return `error` and the test would check if its nil - or skip test with err message